lib/types.ts

```diff
@@ -326,6 +328,13 @@ export namespace OfflinePayment {
   }
 }
 
+export namespace InvoiceWrapper {
```

Comment:
Please don't use namespaces. This is a deprecated pattern in typescript.

Reply:
Thank you, I adjusted to your feedback in: https://github.com/PaymentRails/javascript-sdk/pull/30/commits/08205b5de489061662d087a60fa203f5ddf2266b

This changed:
- Namespaces are no longer used
- `Money` was updated to `Currency`
- `InvoiceWrapper` was updated to a generic `ApiResponse` interface where the consumer can set the type of response expected. This could be reused across other API requests later on if you'd like

Although it has the added benefit of being a generic, I could not get around needing to add:

```typescript
[key: string]: T | boolean;
```

so that `ok: boolean` can be added to the interface definition

Lmk if anything, best!